src/azure-cli/azure/cli/command_modules/resource/_bicep.py

```diff
@@ -177,6 +187,36 @@ def supports_bicep_publish():
     return semver.compare(installed_version, "0.4.1008") >= 0
 
 
+def _is_running_in_ci():
+    if "GITHUB_ACTIONS" in os.environ or "TF_BUILD" in os.environ:
+        return True
+    return False
+
+
+def _use_binary_from_path(cli_ctx):
+    use_binary_from_path = cli_ctx.config.get("bicep", "use_binary_from_path", "if_running_in_ci").lower()
+
+    if use_binary_from_path == "if_running_in_ci":
+        # With if_running_in_ci, GitHub Actions and Azure Pipeline users may expect some delay (usually a few days)
+        # in getting the latest version of Bicep CLI, since the az bicep commands will use the pre-installed Bicep CLI
+        # on the build agents, but the build agents has a different release cycle. The benefit is that the az bicep
```

Comment:
Do you know how release cycles are managed for these build agents? Is there a way we can get visibility into what versions are pre-installed?

Reply:
The releases are visible at https://github.com/actions/runner/releases and https://github.com/microsoft/azure-pipelines-agent/releases. Based on the historical data they usually get published 2 to 3 times per month.